VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -219,7 +219,12 @@ private void bindViewHolderToResponseRow(DiscussionResponseViewHolder holder, fi
         holder.addCommentLayout.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                router.showCourseDiscussionComments(context, comment);
+                if (0 == comment.getChildren().size()) {
```

Comment:
> Also, I noticed that the getChildren() can technically return null

I think that would be considered an API bug. The API is supposed to return an empty list